models/bamf_nnunet_mr_breast/config/default.yml

```diff
@@ -0,0 +1,67 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Default configuration for Bamf NNUnet Breast and tumor segmentation on MR scans (dicom to dicom)
+
+segdb:
+  triplets:
+    T_BREAST_FGT_STRUCTURE:
+      code: C114467
+      meaning: Breast Fibroglandular Tissue
+      scheme_designator: NCIt
+    T_BREAST_CARCINOMA_MASS:
+      code: 4147007
+      meaning: Mass
+      scheme_designator: SCT
+  segments:
+    FGT:
+      name: FGT
+      category: C_RADIOLOGIC_FINDING
+      type: T_BREAST_FGT_STRUCTURE
+      color: [128, 174, 128]
+    BREAST_CARCINOMA:
+      name: Breast_Carcinoma
+      category: C_MORPHOLOGICALLY_ABNORMAL_STRUCTURE
+      type: T_BREAST_CARCINOMA_MASS
+      modifier: M_RIGHT_AND_LEFT
+      color: [144, 238, 144]
+
+execute:
+- DicomImporter
+- NiftiConverter
+- module: NNUnetRunnerV2
+  nnunet_dataset: Dataset009_Breast
+  roi: BREAST+FGT
+- module: NNUnetRunnerV2
+  nnunet_dataset: Dataset011_Breast
+  roi: BREAST+BREAST_CARCINOMA
+- BreastPostProcessor
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  NNUnetRunnerV2:
+    in_data: nifti:mod=mr
+    nnunet_config: 3d_fullres
```

Comment:
Should this parameter be configurable? I believe it's highly specific to the trained model and shouldn't be changed by the user.

Reply:
We do have models in the pipeline where we tried cascade model too. So I left that option there for the future. But as far as this specific model goes, it only has `3d_fullres`. I have removed it from the config for this iteration

Reply:
Okay, either is fine in that case. @rahul99, I now implemented an (experimental) routine to host reusable modules in a separate mhubio extension repository which can then be reused across mhubio modules. Since you use some custom modules for a variety of models, this could be an interesting use-case (like a BAMF-utility-collection). The injection of such mhubio collections is explained [here](https://github.com/rordenlab/dcm2niix). Since we don't have documentation on how to write such collection repositories, we can schedule a call to demonstrate the current approach if that sounds interesting to you.

Reply:
@vanossj , fyi

Reply:
The goal is to get the AIMI PRs finished next week. I would rather not start implementing an experimental feature now.